python/python-core/syntax-and-numerical-operators/python-data-types.md

```diff
@@ -21,44 +21,39 @@ revisionQuestion:
 
 # Fundamental Data types
 
-
 ---
 
 ## Content
 
-Everything in Python has a data type, or *class*[1]. The ones you need to know about now are:
-
-- **Integers** - any whole number. Integers belong to the `int` class:
+Everything in Python has a data type or a [class](https://enki.com/glossary/general/class). The most important ones are as follows:
 
-```bash
->>> type(12)
-# <class 'int'>
-```
+- **Strings** (str) - To store any sequence of characters. Like a name or an address.
 
-- **Floats** - any decimal number. Floats belong to the `float` class:
+A string is always wrapped in either `'single'` or `"double quotes"`
 
 ```bash
->>> type(12.0)
-# <class 'float'>
+>>> type('person123')
+# <class 'str'>
 ```
 
-For all you mathematicians out there, there's one more numerical data type to be aware of: `complex`[2]. For everyone else, you don't need to worry about these 😅!
+Though it's more pythonic[2] to use single quotes until you need double quotes.
```

Comment:
```suggestion
Although it's more pythonic[2] to use single quotes until you need double quotes.
```

I think Although is better for starting a sentence than the shortened though version